apps/buzzer-test/app.c

```diff
@@ -115,7 +115,7 @@ bool app_loop(void) {
             900,
         };
         application_state.play = false;
-        for(size_t i = 0; i < sizeof(rains); i++) {
+        for(size_t i = 0, count = sizeof(rains) / sizeof(rains[0]); i < count; i++) {
```

Comment:
I think this worked before because the ARM C compiler treated each note enum as an `int8_t` due to its size. From my testing, the Emscripten C compiler behaves differently.